base/system/diskpart/diskpart_msg.mc

```diff
@@ -972,39 +974,41 @@ Example:
     CREATE PARTITION EXTENDED SIZE=1000
 .
 Language=Romanian
-    Creates an extended partition on the disk with focus.
-    Applies to master boot record (MBR) disks only.
+    Crează o partiție expandată pe discul în cauză.
```

Comment:
@AndreiMiloiu: As a side comment (not a translation quality review-related one) on "extended partition" - this is not a trivial bit of text. It is a term that I'd suggest you to pay more attention to.
Wikipedia in English describes it here: https://en.wikipedia.org/wiki/Disk_partitioning#Extended_partition
The same Wikipedia, in Romanian, it is translated as "extinsă/extinse": https://ro.wikipedia.org/wiki/Parti%C8%9Bie_(informatic%C4%83)#Tipuri_de_parti%C8%9Bii
You may chose to translate the "extended partition" as "partiție expandată" if you find more fitting to do so, but it's important to be fully aware and intentional about it.